test/special/babel.js

```diff
@@ -98,35 +98,27 @@ describe('babel special parser', () => {
     result.should.deepEqual(['babel-preset-es2015']);
   });
 
-  testCases.forEach(testCase =>
-    it(`should ${testCase.name} in .babelrc file`, () =>
-      testBabel('.babelrc', testCase.deps, testCase.options)));
+  testCases.forEach(testCase => it(`should ${testCase.name} in .babelrc file`, () => testBabel('.babelrc', testCase.deps, testCase.options)));
 
-  testCases.forEach(testCase =>
-    it(`should ${testCase.name} inside .babelrc file env section`, () =>
-      testBabel('.babelrc', testCase.deps, {
-        env: {
-          production: testCase.options,
-        },
-      })));
+  testCases.forEach(testCase => it(`should ${testCase.name} inside .babelrc file env section`, () => testBabel('.babelrc', testCase.deps, {
+    env: {
+      production: testCase.options,
+    },
+  })));
 
-  testCases.forEach(testCase =>
-    it(`should ${testCase.name} in package.json file`, () =>
-      testBabel('package.json', testCase.deps, {
-        name: 'my-package',
-        version: '1.0.0',
-        babel: testCase.options,
-      })));
+  testCases.forEach(testCase => it(`should ${testCase.name} in package.json file`, () => testBabel('package.json', testCase.deps, {
+    name: 'my-package',
+    version: '1.0.0',
+    babel: testCase.options,
+  })));
 
-  testCases.forEach(testCase =>
-    it(`should ${testCase.name} inside package.json file env section`, () =>
-      testBabel('package.json', testCase.deps, {
-        name: 'my-package',
-        version: '1.0.0',
-        babel: {
-          env: {
-            development: testCase.options,
-          },
-        },
-      })));
+  testCases.forEach(testCase => it(`should ${testCase.name} inside package.json file env section`, () => testBabel('package.json', testCase.deps, {
```

Comment:
The airbnb style calls for implicit returns [on the same line](https://github.com/airbnb/javascript#whitespace--implicit-arrow-linebreak) (as seen here). 

However, based on how the tests are written in this repo (with lots of nested implicit returns), that option makes them unreadable, so I think we should change the setting to ["implicit return below"](https://eslint.org/docs/rules/implicit-arrow-linebreak)

P.S. Sorry that this PR is turning into a style / eslint config discussion @nkbt  😞  I know it's a chore (but I agree that it's better to have the linter running / enforced!)

Reply:
No problem, I'll fix that

I would actually prefer to have `prettier` in place and just do not even think about style anymore. Basically as a common style that everyone equally dislikes but since it is automatic and you don't need to care about it - agree to follow =)

Reply:
I turned off that rule since there is a lot of code that follows either way